library/ws-widget/collection/src/lib/profile-v2/profile-certificate-dialog/profile-certificate-dialog.component.ts

```diff
@@ -22,6 +23,7 @@ export class ProfileCertificateDialogComponent implements OnInit {
   author!: string
   userID: any
   courseData: any
+  environment!: any
 
   navUrl: any = ''
   shareUrl = 'https://medium.com/@garfunkel61/angular-simplest-solution-for-social-sharing-feature-6f00d5d99c5e'
```

Comment:
are we using this shareurl?

Reply:
that is existing code, I am not sure we are using that.